lib/commanded/event/handler.ex

```diff
@@ -459,7 +470,7 @@ defmodule Commanded.Event.Handler do
               | :skip
               | {:stop, reason :: term()}
 
-  @optional_callbacks init: 0, init: 1, error: 3
+  @optional_callbacks init: 0, init: 1, error: 3, handle: 2, handle_batch: 2
```

Comment:
users should implement either or. We could throw an error at compile time in the `__using__` macro if we want to

Reply:
I agree it would be useful to throw an error if an event handler module implements both `handle/2` and `handle_batch/2` callback functions.